### ISSUE TITLE
Is it possible to add the experimental options from your StyleGAN2 to this GAN model?

### ISSUE THREAD

**Issue by ilem777** [lucidrains/lightweight-gan#57]
Hello, this GAN you provided is very meaningful for my research as I have a limited set of images (only close to 9,000).
I would like to ask if you have the possibility to add your experimental features in StyleGAN2 （https://github.com/lucidrains/stylegan2-pytorch#experimental） to the training of this GAN model? This could enrich my research.
Your experimental options in StyleGAN2 are these.
1、Top-k Training for Generator
2、Feature Quantization
3、Contrastive Loss Regularization
4、Relativistic Discriminator Loss

**Comment by deklesen** [comment]
I would also love to see this. Especially top-k training and a parameter to increase network capacity.

**Comment by Mut1nyJD** [comment]
@deklesen 

> I would also love to see this. Especially top-k training and a parameter to increase network capacity.

It already those top-k training explicitly for the generator if you look closely in the training loop you can see it does.
